CHANGELOG.md

```diff
@@ -93,6 +93,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Added unbatching logic for `torch.sparse` tensors ([#7037](https://github.com/pyg-team/pytorch_geometric/pull/7037))
 - Added the `RotatE` KGE model ([#7026](https://github.com/pyg-team/pytorch_geometric/pull/7026))
 - Added support for Apple silicon GPU acceleration in some main examples ([#7770](https://github.com/pyg-team/pytorch_geometric/pull/7770), [#7711](https://github.com/pyg-team/pytorch_geometric/pull/7711), [#7784](https://github.com/pyg-team/pytorch_geometric/pull/7784), [#7785](https://github.com/pyg-team/pytorch_geometric/pull/7785))
+- Added `LCMAggregation`, an implementation of the Learnable Communitive Monoid for GNN's ([#7574](https://github.com/pyg-team/pytorch_geometric/issues/7574))
```

Comment:
Move to the top of the `Added` part and associate it with this PR instead of the issue.
```suggestion
- Added `LCMAggregation`, an implementation of the Learnable Communitive Monoid for GNN's ([#7976](https://github.com/pyg-team/pytorch_geometric/pull/7976))
```